### PR TITLE
Axios merge

### PR DESCRIPTION
**Pull request by jacode77**
merged both axios branches so backend is rendering data to the front end

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -18,7 +16,8 @@ import NotFound from "./components/NotFound";
 import { reducer } from "./utils/reducer";
 import { StateContext } from "./utils/stateContext";
 import SignUpForm from "./components/SignUpForm";
-import { getWineListings } from './services/wineServices';
+import { getWineListings } from "./services/wineServices";
```
Comment (NgaNaNa) [inline src/App.js:19]: This line of code is a duplication of what is already in main. We can let is merge to main again and see what happens?

File: src/App.js [hunk 2]
```diff
@@ -31,24 +30,32 @@ function App() {
...
+  // loads initialWineList in componentDidMount
+  useEffect(() => {
+    getWineListings().then((wineListings) => {
       dispatch({
         type: "setWineListings",
-        data: wineListings
-      })
-    })
-    // fetch
+        data: wineListings,
```
Comment (NgaNaNa) [inline src/App.js:52]: I believe you need to remove the comma at end of this line here. Not sure if this makes any difference to your code.

File: src/App.js (hunk 1 above)
Comment (jacode77) [reply src/App.js:19]: I think this is from deleting the line and re-adding it. It's still in the file

**Review by NgaNaNa — APPROVED** [review]
There's two single comments I've made. Otherwise, all looks great!

File: src/App.js (hunk 2 above)
Comment (jacode77) [reply src/App.js:52]: that one automatically adds into the code with the prettier extension

**Pull request merged**